### PR TITLE
add plugins info and dev rules 

### PR DESCRIPTION
**Pull request by tiagotrs**
info copied from plugins descriptions and dev rules from plugins/README

### CODE REVIEW

File: plugins.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Plugins
+=======
+
+The plugin system of Electrum is designed to allow the development of new features without increasing the core code of Electrum. To enable or disable Plugins, see menubar > `Tools` > `Plugins`. 
+
+Below you can find a short description of each available tool. 
+
+
+Audiomoden
```
Comment (SomberNight) [inline plugins.rst:9]: typo: "modem"

**Comment by SomberNight** [comment]
Could you rebase and squash the branch?

**Comment by tiagotrs** [comment]
sorry for that..

**Comment by SomberNight** [comment]
Thanks.

**Pull request merged**